platform/darwin/test/MGLTileSetTests.mm

```diff
@@ -102,6 +102,23 @@ - (void)testTileSetFromTileURLTemplates {
 
     // the scheme is reflected by the mbgl tileset
     XCTAssertEqual(tileSet.scheme, mbgl::Tileset::Scheme::TMS);
+
+    // when the dem enciding is changed using an NSNumber
+    tileSet = MGLTileSetFromTileURLTemplates(tileURLTemplates, @{
+        MGLTileSourceOptionDEMEncoding: @(MGLDEMEncodingTerrarium),
+    });
+
+    // the encoding is reflected by the mbgl tileset
+    XCTAssertEqual(tileSet.encoding, mbgl::Tileset::DEMEncoding::Terrarium);
+
+    // when the dem enciding is changed using an NSValue
+    MGLDEMEncoding terrarium = MGLDEMEncodingTerrarium;
+    tileSet = MGLTileSetFromTileURLTemplates(tileURLTemplates, @{
+        MGLTileSourceOptionDEMEncoding: [NSValue value:&terrarium withObjCType:@encode(MGLDEMEncoding)],
```

Comment:
Use `@(MGLDEMEncoding)` instead, so that the value in the dictionary is “boxed” in an `NSInteger` instead of an `NSValue`. Most option dictionaries use `NSInteger`s for more convenient unboxing.

Reply:
Should I delete this test altogether then because the one above it uses `@(MGLDEMEncoding)` and NSNumber

Reply:
Oh, I see, I misread this. You’re right, carry on. 👍